### ISSUE TITLE
Webview showing only white screen after upgrading to 5.1.0+4

### ISSUE THREAD

**Issue by rahuldevanavar91** [pichillilorenzo/flutter_inappwebview#736]
- [x] I have read the [README](https://github.com/pichillilorenzo/flutter_inappwebview/blob/master/README.md)
- [x] I have already searched for the same problem
- [x] I have read and done the setup for Android: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for Android](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android)
- [x] I have read and done the setup for iOS: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-ios)
- [x] I have ran the sample app and it does not work there

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |          2.0.2      |
| Plugin version       |         5.1.0+4      |
| Android version      |           30    |

Device information: 
One plus 5 

## Description

**Expected behavior:** 
webview should work for all cases 

**Current behavior:** 
after  upgrading to plugin 5.1 webview is not loading properly ..specially when we have the multiple  webview pages  its showing bank .if I downgrade to 4.0.0+4 its working fine .

## Steps to reproduce
One the below code click on webview 1
in webview right hamburger menu click on login or do some actions so that new pages will be opened 
now back press and click on webview 2 ..
it will show only blank page .

```
import 'package:flutter/material.dart';
import 'package:flutter_inappwebview/flutter_inappwebview.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);
  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            InkWell(
              onTap: () {
                Navigator.of(context).push(MaterialPageRoute(
                    builder: (context) =>
                        WebviewPage("https://www.acko.com/")));
              },
              child: SizedBox(height: 50, width: 100, child: Text("Webview 1")),
            ),
            InkWell(
              onTap: () {
                Navigator.of(context).push(MaterialPageRoute(
                    builder: (context) =>
                        WebviewPage("https://www.amazon.com/")));
              },
              child: SizedBox(height: 50, width: 100, child: Text("Webview 2")),
            )
          ],
        ),
      ),
    );
  }
}

class WebviewPage extends StatefulWidget {
  String url;

  WebviewPage(this.url);

  @override
  WebviewPageStete createState() => WebviewPageStete();
}

class WebviewPageStete extends State<WebviewPage> {
  @override
  Widget build(BuildContext context) {
    // TODO: implement build
    return InAppWebView(
      initialUrlRequest: URLRequest(url: Uri.parse(widget.url)),
    );
  }
}

```

## Stacktrace/Logcat   
```

2021-03-22 15:29:47.652 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.700 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.739 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.788 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.815 23681-23692/com.test.flutter_module_test.host I/zygote: NativeAlloc concurrent copying GC freed 11(47KB) AllocSpace objects, 0(0B) LOS objects, 50% free, 1543KB/3MB, paused 6.513ms total 43.325ms
2021-03-22 15:29:47.880 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.911 23681-23692/com.test.flutter_module_test.host I/zygote: NativeAlloc concurrent copying GC freed 38(48KB) AllocSpace objects, 0(0B) LOS objects, 49% free, 1559KB/3MB, paused 8.007ms total 46.704ms
2021-03-22 15:29:47.920 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.927 23681-23692/com.test.flutter_module_test.host I/zygote: WaitForGcToComplete blocked NativeAlloc on None for 5.499ms
2021-03-22 15:29:47.974 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:47.985 23681-23692/com.test.flutter_module_test.host I/zygote: NativeAlloc concurrent copying GC freed 12(32KB) AllocSpace objects, 0(0B) LOS objects, 49% free, 1559KB/3MB, paused 6.683ms total 57.111ms
2021-03-22 15:29:48.048 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.094 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.135 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.171 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.171 23681-23692/com.test.flutter_module_test.host I/zygote: NativeAlloc concurrent copying GC freed 10(47KB) AllocSpace objects, 0(0B) LOS objects, 50% free, 1560KB/3MB, paused 7.320ms total 45.926ms
2021-03-22 15:29:48.211 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.278 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.318 23681-23681/com.test.flutter_module_test.host W/zygote: Attempt to remove non-JNI local reference, dumping thread
2021-03-22 15:29:48.338 23681-23692/com.test.flutter_module_test.host I/zygote: NativeAlloc concurrent copying GC freed 11(32KB) AllocSpace objects, 0(0B) LOS objects, 49% free, 1559KB/3MB, paused 6.920ms total 67.818ms

```

**Comment by pichillilorenzo** [comment]
Tested on a real Android device and on Android Emulator API 29 and API 30 and it works correctly in all cases.
Are you sure you don't have internet connection problems?

However sometimes seems the https://www.acko.com/ website takes a long time to load the webpage (in one case I have also got `ERR_CONNECTION_CLOSED`, so maybe the https://www.acko.com/ website is having some issues).

Here is a screen recording of the real Android device:

https://user-images.githubusercontent.com/5956938/111976738-40e6e000-8b02-11eb-942b-364bf815aec1.mp4

As you can see, it works correctly every time. But, as already said, sometimes, the https://www.acko.com/ website takes a long time to load.

**Comment by rahuldevanavar91** [comment]
I m checking on emulator 
Pixel 4 api 27.

**Comment by rahuldevanavar91** [comment]
I am checking on emulator 
Api 27 Google pixel 4

**Comment by rahuldevanavar91** [comment]
@pichillilorenzo I am checking in emulator API 27 PIXEL 4.


Screen recording for 5.1.0+4

https://user-images.githubusercontent.com/12570474/112001082-9c39c200-8b44-11eb-89f4-9889cfaad057.mov

Screen recording for 4.0.0+

https://user-images.githubusercontent.com/12570474/112001090-a0fe7600-8b44-11eb-8920-aa89d6bd6acc.mov

**Comment by pichillilorenzo** [comment]
Tested on Android Emulator API 27 PIXEL 4 using `5.1.0+4` and it works just fine.

Here is the screen recording:

https://user-images.githubusercontent.com/5956938/112006966-87e5cd00-8b24-11eb-99e5-e50ae75e016d.mp4

I have updated your `WebviewPageStete` class with some events in order to log info about the loading state of the webpage:
```dart
class WebviewPageStete extends State<WebviewPage> {
  @override
  Widget build(BuildContext context) {
    return InAppWebView(
      initialUrlRequest: URLRequest(url: Uri.parse(widget.url)),
      onLoadStart: (controller, url) {
        print("LOAD START $url");
      },
      onProgressChanged: (controller, progress) {
        print("Loading $progress%");
      },
      onLoadStop: (controller, url) {
        print("LOAD STOP $url");
      },
    );
  }
}
```
Try it and check the logs.

Probably, you have some issue with your internet connection because I don't have any problem loading https://www.acko.com/ using the same emulator or the website itself is having problems.
If it was the `InAppWebView` plugin, then also https://www.amazon.com/ website should be just a white screen but it isn't.
So, or you have some internet connection issues or the website itself takes a lot of time to load (sometimes).
